src/server/parser.js

```diff
@@ -0,0 +1,317 @@
+/*jslint devel: true, nomen: true, plusplus: true, regexp: true, indent: 2, maxlen: 100 */
+
+"use strict";
+
+var logger = require('./logger');
+var async = require('async');
```

Comment:
Is this going to cause problems with newer JS interpreters since "async" is a reserved keyword?